docs/pages/v2/crm-v2/crm-v2.yaml

```diff
@@ -26,12 +26,16 @@ paths:
                     type: string
                   userId:
                     type: string
+                  currency:
+                    type: string
+                    description: The default currency for the organization in ISO format
```

Comment:
should we note this will be null for pipedrive since the curency live son the deal